moveit_py/src/moveit/moveit_py_utils/CMakeLists.txt

```
@@ -13,4 +13,4 @@ install(
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin)
 
-install(DIRECTORY include/ DESTINATION include)
+install(DIRECTORY include/ DESTINATION include/moveit_py_utils)
```

I'm sorry I made a mistake in what I suggested here. The package is `moveit_py`.

```suggestion
install(DIRECTORY include/ DESTINATION include/moveit_py)
```